deploy/kubernetes/overlays/linux/base/volumes.yaml

```diff
@@ -0,0 +1,44 @@
+kind: DaemonSet
```

Comment:
Can this be combined with gcepd.yaml?

Reply:
we could, but from kustomization doc, they suggested to make small changes for each file, since volumes and gcepd container are two different parts, I use two files.

Reply:
I think it makes sense to keep independent patches in separate files, but in this case, since container.volumeMounts and pod.spec.volumes are tied together, I think it makes sense to keep them together.

Reply:
got it. I updated it.